plugins/importexport/native/filter/NativeXmlPKPAuthorFilter.inc.php

```diff
@@ -77,11 +79,11 @@ function handleElement($node) {
 		if ($node->getAttribute('seq')) $author->setSequence($node->getAttribute('seq'));
 
 		// Identify the user group by name
-		$userGroupName = $node->getAttribute('user_group_ref');
+		$userGroupName = Stringy::create($node->getAttribute('user_group_ref'));
 		$userGroupDao = DAORegistry::getDAO('UserGroupDAO'); /** @var $userGroupDao UserGroupDAO */
 		$userGroups = $userGroupDao->getByContextId($context->getId());
 		while ($userGroup = $userGroups->next()) {
-			if (in_array($userGroupName, $userGroup->getName(null))) {
+			if ($userGroupName->containsAny($userGroup->getName(null) ?? [], false)) {
```

Comment:
I just applied this "fix" here, perhaps we should adopt case-insensitive comparisons on any generic string key 🤔

Another option, that would also address the value validation implicitly, would be to build the DTD on run-time, so we would be able to add value requirements to the attributes.

Reply:
Considering we're looking forward to replacing the plugin by the API, I'll ignore my suggestion here =]